source/features/embed-gist-via-iframe.tsx

```diff
@@ -0,0 +1,37 @@
+import select from 'select-dom';
+import features from '../libs/features';
+
+function init(): void {
+	const embedViaScript = select('.file-navigation-option .select-menu-item')!;
```

Comment:
Still too broad; how about:

```suggestion
	const embedViaScript = select('.file-navigation-option [value$="<script"]')!;
```

Reply:
I guess you are writing `$('.file-navigation-option [value^="<script"]')` 😂